### PR TITLE
AF-1892 Dart 2 compatibility

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Description

Make sockjs_client_wrapper compatible with Dart 2.

## Testing

- [ ] CI passes (dart 1.24.3 and dev channel)

## Code Review
@Workiva/app-frameworks 
@robbecker-wf

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on HipChat: InfoSec Forum._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.5
+
+- Dart 2 compatible.
```
Comment (robbecker-wf) [inline CHANGELOG.md]: 👏

**Comment by robbecker-wf** [comment]
+10 CI passes under dart 1 and dart 2

**Comment by robbecker-wf** [comment]
Quality Review Approval: +1 ( aka QA: +1 )

- [x] Testing instruction
- [x] Dev +1's
- [x] Dev/QA +10 with detail of what was tested
- [ ] Unit test created/updated (NA)
- [x] All unit tests pass
- [x] Rosie has run and reports properly the release the ticket will be included in

Merging into master.
@Workiva/release-management-p for merge into master

**Pull request merged**